src/components/card-add/ExpirationDateInput.jsx

```diff
@@ -0,0 +1,47 @@
+import { MONTH, YEAR } from "../../constants/expirationDate";
+import { getNumberString } from "../../util/regExp";
+
+export default function ExpirationDateInput({
+  expirationDate,
+  setExpirationDate,
+}) {
+  const onChangeExpirationDate = (event) => {
+    const { name, value } = event.target;
+    const onlyNumberValue = getNumberString(value);
+
+    // 두 자리수 초과
+    if (onlyNumberValue.length > 2) {
+      return;
+    }
+
+    // 유효하지 않은 월
+    if (name === MONTH && +onlyNumberValue > 12) {
+      return;
+    }
```

Comment:
날짜 값이 유효한지 검사하는 로직은 유틸 함수로 분리해도 좋을 것 같아요.

```suggestion
    try {
      if (isValidDate({ month: ..., day: ... })) {
        throw new Error(INVALID_DATE);
      }
    } catch() {
      return; // 혹은 예외 처리
    }
```